corehq/apps/es/aggregations.py

```diff
@@ -102,7 +102,7 @@ def buckets_dict(self):
 
     @property
     def buckets_list(self):
-        return {Bucket(b, self._aggregations) for b in self.normalized_buckets}
+        return [Bucket(b, self._aggregations) for b in self.normalized_buckets]
```

Comment:
I think it should be good to merge. I did a quick look for where this was used and think that failed test was the only change needed.